modules/ROOT/pages/migration/index.adoc

```diff
@@ -18,533 +18,56 @@ npm update @neo4j/graphql
-The following implicit equality filters are deprecated: 
-
-  - `{ name: "Keanu" }`
-  - `{ count: 10 }` 
-
-You can achieve the same by using `{ name_EQ: "Keanu" }` and `{ count_EQ: 10 }`.
-The deprecated quality filters will be removed in version 7.x.
-
-[cols="1,1"]
-|===
-|Before | Now
-
-a|
-[source, graphql, indent=0]
-----
-query {
-  users(where: { name: "John" }) {
-    id
-    name
-  }
-}
-----
-a|
-[source, graphql, indent=0]
-----
-query {
-  users(where: { name_EQ: "John" }) {
-    id
-    name
-  }
-}
-----
-|===
-
-=== `@node` will have to be explicitly defined
-
-In the future, types without the `@node` directive will no longer be treated as Neo4j nodes.
-In version 6.0.0, it's not required to specify every GraphQL type representing a Neo4j node with the `@node` directive, however it's recommended to do so and a warning is raised if you don't.
-There are genuine cases in which a `@node` should not be used, e.g., for types returned by @cypher fields which are not Neo4j nodes.
-
-[cols="1,1"]
-|===
-|Before | Now
-
-a|
 [source, graphql, indent=0]
 ----
 type Movie {
-    title: String
-    actors: [Person!]! @relationship(type: "ACTED_IN", direction: IN)
-}
-
-type Person {
-    name: String
+    title: String! @unique()
 }
 ----
-a|
-[source, graphql, indent=0]
-----
-type Movie @node {
-    title: String
-    actors: [Person!]! @relationship(type: "ACTED_IN", direction: IN)
-}
 
-type Person @node {
-    name: String
-}
-----
-|===
 
-=== Deprecated the implicit equality filters
+=== Non-list relationships
 
-Previously, if a field was present in a filter without specifying the operator, it was treated as an equality filter.
-This behavior is now deprecated and will be removed in the future.
-Use the `_EQ` filter instead.
+Deprecate single element relationships:
 
-[cols="1,1"]
-|===
-|Before | Now
 
-a|
-[source, graphql, indent=0]
-----
-{
-  movies(where: { title: "The Matrix" }) {
-    title
-  }
-}
-----
-a|
 [source, graphql, indent=0]
 ----
-{
-  movies(where: { title_EQ: "The Matrix" }) {
-    title
-  }
+type Movie {
+    director: Person @relationship(type: "DIRECTED", direction: "IN")
 }
 ----
-|===
-
-=== Deprecated pagination `options` argument
 
-The `options` argument in query and `@relationship` fields is deprecated and will be removed in the future.
-Use the `limit`, `offset` and `sort` arguments instead.
+In favor of list relationships:
 
-[cols="1,1"]
-|===
-|Before | Now
 
-a|
-[source, graphql, indent=0]
-----
-{
-  movies(options: { limit: 10, offset: 10, sort: { title: ASC } }) {
-    title
-  }
-}
-----
-a|
 [source, graphql, indent=0]
 ----
-{
-  movies(limit: 10, offset: 10, sort: { title: ASC }) {
-    title
-  }
+type Movie {
+    director: [Person!]! @relationship(type: "DIRECTED", direction: "IN")
 }
 ----
-|===
 
+1-1 relationships cannot be reliably enforced, leading to a data inconsistent with the schema. For this reason, these have been removed in favor of the more accurate list relationships.
```

Comment:
```suggestion
Single element relationships cannot be reliably enforced, leading to a data inconsistent with the schema.
```

what does "enforced" mean? sounds like there is a technical limitation in graphql?